python/requirements.txt

```diff
@@ -0,0 +1 @@
+../requirements.txt
```

Comment:
Is this relative path necessary because the package needs a `requirements.txt` in the same directory as its config file?

Reply:
Good question! This isn't necessary, but I thought it might be helpful, since it's standard practice to include some sort of dependency file in the root of a Python project. In this repo, the `python/` subdir acts as the home for the Python project, but `requirements.txt` needs to be defined in the root of the repo so that reticulate can understand it. Adding a symlink to `requirements.txt` from `python/requirements.txt` just helps provide a pointer to any code readers who may be interested in inspecting the Python package in isolation. Does that make sense?

Reply:
Yep, totally makes sense.